ingester/user_state.go

```diff
@@ -28,10 +29,11 @@ type userState struct {
 	ingestedSamples *ewmaRate
 }
 
-func newUserStates(rateUpdatePeriod time.Duration) *userStates {
+func newUserStates(rateUpdatePeriod time.Duration, maxSeriesPerUser int) *userStates {
```

Comment:
+1 UserStatesConfig - I suspect we're going to have more shortly (to limit the number of series for a single metric name, for instance)